src/drive/web/modules/navigation/Breadcrumb/breadcrumb.styl

```diff
@@ -26,6 +41,9 @@
     .fil-path-separator
         margin 0 .25rem
 
+    &:hover
+        color          var(--black)
```

Comment:
nit: We should use semantic variable. But maybe for this one, this is a special case when we don't want to theme?

Reply:
i used `primaryTextColor`